examples/plotting_with_folium.ipynb

```diff
@@ -0,0 +1,519 @@
+      "text/plain": [
+       "<Figure size 432x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df.plot(figsize=(6, 6))\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Project geometry to WGS84 (EPSG4326)"
```

Comment:
Can you explain in the text why are you reprojecting it?

Reply:
added explanation!